File: python/week-01/day03_bmi.py
"""
Day: 03
Date: 2026-08-03
Topic: Calculate BMI (Body Mass Index)
Challenge: Take weight and height as input and display the BMI and Print the BMI and the category
What I Learned: [Fill this AFTER completing]
Difficulty: [Easy]
Time Taken: [10 minutes]
"""

# Your code here
weight = float(input("Enter your weight in kg: "))
height = float(input("Enter your height in meters: "))
bmi = weight / (height ** 2)
print(f"Your BMI is: {bmi:.2f}")

if bmi < 18.5:
    category = "Underweight"
elif 18.5 <= bmi < 25:
    category = "Normal weight"
elif 25 <= bmi < 30:
    category = "Overweight"
else:
    category = "Obese"

print(f"Category: {category}")

# ============ NOTES ============
# Write any observations, tricks, or "aha!" moments here

File: cpp/week-01/day03_bmi.cpp
/*
 * Day: 03
 * Date: 2026-08-03
 * Topic: Calculate BMI (Body Mass Index)
 * Challenge: Take weight and height as input and display the BMI and Print the BMI and the category
 * What I Learned: [Fill this AFTER completing]
 * Difficulty: [Easy]
 * Time Taken: [10 minutes]
 */

#include <iostream>
using namespace std;

int main()
{
    // Your code here
    double weight, height, bmi;

    cout << "Enter weight (kg): ";
    cin >> weight;

    cout << "Enter height (m): ";
    cin >> height;

    bmi = weight / (height * height);

    cout << "BMI: " << bmi << endl;

    if (bmi < 18.5)
    {
        cout << "Category: Underweight" << endl;
    }
    else if (bmi >= 18.5 && bmi < 24.9)
    {
        cout << "Category: Normal" << endl;
    }
    else if (bmi >= 25 && bmi < 29.9)
    {
        cout << "Category: Overweight" << endl;
    }
    else
    {
        cout << "Category: Obese" << endl;
    }

    return 0;
}

// ============ NOTES ============
// Write any observations, tricks, or "aha!" moments here